controller/search.go

```diff
@@ -243,3 +245,43 @@ func (c *SearchController) Users(ctx *app.UsersSearchContext) error {
 
 	return ctx.OK(&response)
 }
+
+// Iterate over the WI list and read parent IDs
+// Fetch and load Parent WI in the included list
+func (c *SearchController) enrichWorkItemList(ctx *app.ShowSearchContext, res *app.SearchWorkItemList) {
+	// Need a map of string to UUID to prevent duplicates in place.
+	visitedIDs := map[string]uuid.UUID{}
+	for _, wi := range res.Data {
+		cond := wi.Relationships != nil && wi.Relationships.Parent != nil && wi.Relationships.Parent.Data != nil
+		if cond {
+			parentIDStr := *wi.Relationships.Parent.Data.ID
+			if _, exist := visitedIDs[parentIDStr]; exist == false {
```

Comment:
Instead of `exist == false` you may use `!exist`

Reply:
thanks @baijum done in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/647af81c90f87c8232c1fdd69eaee5cdc11eb496